super_calendar_analytic/README.rst

```diff
@@ -0,0 +1,58 @@
+Features
```

Comment:
Please follow the OCA template:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst